tests/dist-check.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
I chose the name intentionally not to start with `test_` in order for it not to be automatically collected by pytest and auto-collected. I'll update the module with a docstring explaining why.